app/MMB/src/scenes/timeline/TimelineEventIcon.js

```diff
@@ -4,9 +4,13 @@ import * as React from 'react';
 import { View } from 'react-native';
 import { StyleSheet, Color, Icon, TextIcon } from '@kiwicom/mobile-shared';
 
+export const TimelineEventIconContext = React.createContext({
+  displayTopLine: true,
+  displayBottomLine: true,
+});
+
```

Comment:
What is this context used for? 
You set two booleans, why can it not be variables?

Reply:
This is to indicate default values. But that context is set depending on the index of the rendered event components.